locksmith/src/routes/v2/settings.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express'
+import { LockSettingController } from '../../controllers/v2/lockSettingContoller'
+import { authenticatedMiddleware } from '../../utils/middlewares/auth'
+import { lockManagerMiddleware } from '../../utils/middlewares/lockManager'
+
+const router = express.Router({ mergeParams: true })
+
+const lockSettingController = new LockSettingController()
+
+router.post(
+  '/:network/locks/:lockAddress',
+  authenticatedMiddleware,
+  lockManagerMiddleware,
+  (req, res) => {
+    lockSettingController.updateSettings(req, res)
+  }
+)
+
+router.get('/:network/locks/:lockAddress', (req, res) => {
+  lockSettingController.getSettings(req, res)
+})
```

Comment:
Do we want anyone to get lock settings? I assume we will have more options added. I would make it authenticated and only allow lock manager to see.

Reply:
Added `DEFAULT_LOCK_SETTINGS`